packages/run-protocol/jsconfig.json

```diff
@@ -15,5 +15,10 @@
     "strictNullChecks": true,
     "moduleResolution": "node",
   },
-  "include": ["src/**/*.js", "test/**/*.js", "exported.js", "globals.d.ts"],
+  "include": [
+    "src/**/*.js",
+    "test/**/*.js",
+    "exported.js",
+    "globals.d.ts",
```

Comment:
This is reformatted only because I tried an experiment of adding `"bundles/install-on-chain.js" to this list, so it would no longer be exempt from linting.